src/Raven.Studio/typescript/commands/maintenance/monitorCompactCommand.ts

```diff
@@ -7,7 +7,7 @@ class monitorCompactCommand extends commandBase {
     }
 
     execute(): JQueryPromise<any> {
-        new getDocumentWithMetadataCommand("Raven/Database/Compact/Status/" + this.dbName, null)
+        new getDocumentWithMetadataCommand("Raven/Database/Compact/Status/" + this.dbName, null) //TODO: obsolete?
```

Comment:
I'm not sure, right now I don't think it is supported at all, at any rate, it wouldn't be using this doc.